css/index.css

```diff
@@ -143,4 +180,109 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* HOME PAGE BOXES STYLES */
```

Comment:
Great job on the comments, and targeting each box without rewriting the same class over and over again